docs/sql/reference/select.txt

```diff
@@ -91,15 +91,34 @@ FROM Clause
 
 The FROM clause specifies the source table for the SELECT::
 
-    FROM table_ident [ [AS] table_alias ]
+    FROM table_reference [{CROSS JOIN | , } table_reference [{CROSS JOIN | ,} ...]]
 
-:table_ident: The name (optionally schema-qualified) of an existing table.
+Table Reference
+...............
+
+A table Reference is a table ident with an optional table alias::
+
+    table_ident [ [AS] table_alias ]
+
+:table_ident: The name (optionally schema-qualified) of an existing table
```

Comment:
I prefer `.` at the end of descriptions, even if they are not full sentences. geschmacksache ...